meshnets/data_processing/torch_utils.py

```diff
@@ -0,0 +1,64 @@
+"""Utility functions for usage with pytorch"""
+import torch
+import torch_geometric
+
+
+def dict_to_geometric_data(examples,
+                           x_key='node_features',
+                           edge_index_key='edges',
+                           edge_attr_key='edge_features',
+                           y_key='wind_pressures'):
+    """Collate function for pytorch dataloader
+
+    Args:
+        examples (list): List of examples from the dataset
+        x_key (str, optional): Key for the node features. Defaults to
+        'node_features'.
+        edge_index_key (str, optional): Key for the edge
+        indices. Defaults to 'edges'.
+        edge_attr_key (str, optional): Key for the edge
+        features. Defaults to 'edge_features'.
+        y_key (str, optional): Key for the target values. Defaults to
+        'wind_pressures'.
+
+    Returns:
+        torch_geometric.data.Batch: Batch object containing the
+
+    This assumes examples have the structure:
+    {
```

Comment:
Should we try to check for this structure then? Or it will through an error immediately?

Reply:
It will fail pretty quickly